### PR TITLE
Update AddCommand help message and UG for clarity

### LINKED ISSUE #106: Wrong error message for AddCommand
image

This command is copied from the help message.

### LINKED ISSUE #125: Blank background in upcoming interviews window
Just a small aesthetic issue. There's blank background at the bottom of the upcoming interviews window. Probably can add background color.

image

### PR DESCRIPTION
**Pull request by jolynloh**
- Update UG to inform users of 'no space' requirement of tags
- Update UG to inform users of the need to close reminder window to refresh list of upcoming interviews (it used to update without having to close the window but not anymore 😢 so adding this as a stopgap measure until I manage to fix it in the coming weeks) 
- Update help message for AddCommand
- Add background colour to upcoming interviews window

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -307,9 +308,10 @@ Format: `remind`
 * Archived interviews will not be displayed even if they fall within the next week.
 * If there are no upcoming interviews, a blank window will be displayed.
 * `remind` can only be used on the main application list page (i.e. not to be used on list-archive page).
+* To refresh the list of upcoming interviews, close the reminder window (if it is currently opened) and input `remind` command again.
 
-### Clearing all entries: `clear`
-Clears all entries from CinternS.
+### Clearing all entries: `clear` 
+Clears all entries from CinternS.
```
Comment (teekaytai) [inline docs/UserGuide.md:314]: Is this intended? I'm not sure what's this is for

**Review by teekaytai — COMMENTED** [review]
LGTM! Just one question

File: docs/UserGuide.md (hunk 1 above)
Comment (jolynloh) [reply docs/UserGuide.md:314]: oops yea this is not intended 😅 must have edited it accidentally

**Review by nmtuan2001 — COMMENTED** [review]
LGTM!

**Pull request merged**